nautobot/core/templates/generic/object_list.html

```diff
@@ -20,14 +55,47 @@
     {% endif %}
 </div>
 <h1>{% block title %}{{ content_type.model_class|meta:"verbose_name_plural"|bettertitle }}{% endblock %}</h1>
+{% if filter_params %}
+<div class="filters-applied">
+    <b>Filters:</b>
+    {% for field, values in filter_params %}
+    <span class="filter-container display-inline" dir="ltr">
+        <span class="selection">
+        <span
+                class="filter-selection filter-selection--multiple p-5">
+            <b>{{ field }}:</b>
+            <span
+                    class="filter-selection__clear remove-filter-param m-0"
```

Comment:
Why do we need three separate classes here? Can we remove `filter-selection__clear` and `m-0` and just apply all CSS styling to the `remove-filter-param` class?